wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -14,15 +14,19 @@
     - `action_icon` - icon for the 'action' button, default is 'icon-plus'
 
 {% endcomment %}
-<header class="header {% if merged %}merged{% endif %} {% if search_form %}hasform{% endif %}">
+<header class="header{% if merged %} merged{% endif %}{% if search_form %}hasform{% endif %} {{ classname }}">
     {% block breadcrumb %}{% endblock %}
     <div class="row">
         <div class="left">
             <div class="col">
                 <h1 class="header__title">
-                    {% if icon %}{% icon name=icon %}{% endif %}
+                    {% if icon %}
+                        {% icon class_name="w-header__icon" name=icon %}
+                    {% elif avatar %}
+                        <div class="avatar"><img src="{% avatar_url user %}" alt="" /></div>{% endif %}
```

Comment:
```suggestion
                        <div class="avatar"><img src="{% avatar_url user %}" alt="" /></div>
                    {% endif %}
```
Small nitpick but try to keep the closing endif on the same line indentation as the if opening.

Reply:
Remember to add that the template accepts an avatar url in the template description at the top.